domains/albin.json

```
@@ -0,0 +1,11 @@
+ 
+    {
+      "owner": {
+        "username": "ALBINPRAVEEN",
+        "email": "albinpraveen135790@gmail.com"
+      },
+      "record": {
+        "CNAME": "albinpraveen.vercel.app"
```

Sorry, we don't support Vercel as a CNAME but you can redirect to it.

```suggestion
        "URL": "https://albinpraveen.vercel.app"
```